aten/src/ATen/native/quantized/cpu/qnnpack/BUILD.buck

```diff
@@ -0,0 +1,143 @@
+load("//tools/build_defs:glob_defs.bzl", "subdir_glob")
```

Comment:
This is fine for now, but let's target moving these rules to a .bzl file that we can share with the internal build.